src/Resolver/InjectionInterface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-di for the canonical source repository
```

Comment:
@weierophinney do we need to keep these in our CS?

Reply:
Used to be the default header. Has this changed?

Reply:
That's what I was wondering - overall, these copyright notices don't really do much anyway

Reply:
The CS is currently undergoing so much changes. I'd suggest to tackle this in a separate PR when there is a final commitment on the CS. Are you ok with that?

Reply:
:+1:

Reply:
@Ocramius We've not yet finalized the idea of a COPYRIGHT.md file, so, at this point, stick to what we're already doing (which is per-file docblocks).